Library/Homebrew/os/mac/version.rb

```diff
@@ -1,12 +1,13 @@
 # frozen_string_literal: true
 
+require "hardware"
 require "version"
 
 module OS
   module Mac
     class Version < ::Version
       SYMBOLS = {
-        big_sur:     "10.16",
+        big_sur:     Hardware::CPU.arm? ? "11.00" : "10.16",
```

Comment:
Is it really `11.00` and not `11.0`?

Reply:
Opened PR https://github.com/Homebrew/brew/pull/7810

Reply:
after the final release, I think it's all 11.0 now, isn't it?